### ISSUE TITLE
[rush] Add possibility to disable mail prompt for the rush change command

### ISSUE THREAD

**Issue by btmnk** [microsoft/rushstack#2523]
## Details

**What do you think is the cause of this problem?**
When executing the rush change command the CLI will pull the mail information from somewhere (I'm not even sure from where) and requires the user to either accept the proposed mail or change it. There is no way to generally disable this when it's not wanted at all.
In our case for example we are not allowed to have developers personal information in the source code, that also includes the business mail. When running the rush change command and you forget to change the proposed mail it will be written in the change files. In the worst case scenario it's accidentally committed as well and stays there in the source code until the next publish.

**How do you think we should fix this?**
It should be possible to globally disable the mail prompt or set a custom default (e.g. a non personal mailbox).
I think making it possible to disable it would be the easiest approach for now.

## Standard questions

Please answer these questions to help us investigate your issue more quickly:

| Question | Answer |
| -------- | -------- |
| `@microsoft/rush` globally installed version? | 5.39.0 |
| `rushVersion` from rush.json? | 5.39.0 |
| `useWorkspaces` from rush.json? | no |
| Operating system? | Mac |
| Would you consider contributing a PR? | No (probably no time currently) |
| Node.js version (`node -v`)? | 12.18.2 |

**Comment by iclanton** [comment]
Rush gets that from the Git configured `user.email`, which ends up attached to the metadata of every Git commit a user makes (so that's already in your codebase, assuming you're using Git).

You can set the [`gitPolicy.allowedEmailRegExps`](https://rushjs.io/pages/maintainer/setup_policies/#allowedemailregexps-avoiding-private-e-mail-addresses) property in `rush.json` to enforce that users have their email address in git set to an address on a particular domain to avoid personal email addresses.

**Comment by btmnk** [comment]
@iclanton I'm not sure why they have these requirements, of course you're right, with git the users mail is already comitted. I might be able to challenge these requirements and maybe in the end it's not that big of a deal but still knowing that we can prevent the mails from being written in the changefiles if necessary would be reassuring.

Nevertheless I think this would be an DX improvement even if it's not tied to corporation policies as in my case.

**Comment by iclanton** [comment]
@octogonz is going to start working on redesigning publishing soon. He should be able to incorporate this suggestion into that work.

**Comment by octogonz** [comment]
Given that the Rush Stack tooling does not use these emails itself, maybe `rush change` prompting for emails should be off-by-default.  It's easy to implement.  The only interesting question is where to put the config file setting that controls this.

**Comment by iclanton** [comment]
It could go in `version-policies.json` since that controls some other "rush change" related stuff.

**Comment by octogonz** [comment]
> 
> It could go in `version-policies.json` since that controls some other "rush change" related stuff.

I agree.  There's a minor design gap with this approach:  If someone relied on the email behavior, but they use `shouldPublish=true` instead of version policies, then they can't use `version-policies.json` to restore the old behavior.  However I'm okay with that for a couple reasons:

1. I suspect that very few Rush repos rely on these email addresses.

2. The version policies are intended to be a complete replacement for `shouldPublish=true`, so it should be easy to create a version policy so that the setting can be enabled.
